src/client/components/ProductsPagination/ProductsPagination.component.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import ReactPaginate from 'react-paginate';
```

Comment:
You are using ReactPaginate, but you didn't install it (npm install to include it as part of your pacakge.json deps)

beside that, before picking a library to use in the project, a discussion should be helpd about it between the project members (mentors & students)